src/data/faq.json

```diff
@@ -1626,6 +1626,16 @@
                                     "If you continue to receive the error message and have been able to verify that the certificate is indeed not already in your app, please reach out to us by mail at <a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>. If your certificate has been issued incorrectly, please contact the issuer of the certificate."
                                 ]
                             },
+                            {
+                                "title": "Error 'DCC_RI_429'",
+                                "anchor": "dcc_ri_429",
+                                "active": false,
+                                "textblock": [
+                                    "The error message with the code \"DCC_RI_429\" occurs if you try to reissue a certificate more than three times. This could happen, for example, if you want to have the certificate reissued on multiple devices. In this case, you can scan the QR code of the certificate from another device if you have already received the new, replaced certificate there.",
+                                    "Because of this limitation, we recommend that you export the newly issued certificate as a PDF file and print it out so that you can scan the QR code of the certificate again if needed",

```

Comment:
I suggest to add a link to the FAQ entry explaining how to create certificate printouts.

Reply:
excellent idea!

Reply:
link added.